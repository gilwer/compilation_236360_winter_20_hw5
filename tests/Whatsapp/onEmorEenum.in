enum Days {DOMINGO, LUNES,MARTES, MIERCOLES, JUEVES,VIERNES,SABADO};
enum StudyDays {STUDYDAY, ANOTHERSTUDYDAY};

int foof(enum Days d,int z,enum StudyDays sd) {
	printi(42);
	return (int)(d)+1;
}

int foo(int x, int y, int z) {
	return y;
}
int g(enum Days d){
	return (int)d;
}
void main() {
	enum Days d = DOMINGO;
	int x = 10;
	enum StudyDays sd = ANOTHERSTUDYDAY;
	printi(foof(MARTES, foof(d, x, ANOTHERSTUDYDAY), sd));
	printi(foo(x, foo(2, x, x) ,5));
	printi((int)VIERNES);
	printi(g(DOMINGO));
	return;
}