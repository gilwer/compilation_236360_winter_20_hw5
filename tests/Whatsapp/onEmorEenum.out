42
42
3
10
5
0
